bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,23 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.7.0
+************
+
+* Major rewrite of lower transport layer to fix issue where sometimes packages where lost in high bandwidth application.
+* Fixed bug in GPS socket driver which made it some times try to free the same memory twice.
+* Fixed bug where TCP/IP session would sometimes hang when the transfer is done.
+* Allocating more sockets than available will not return ENOBUF instead of ENOMEM.
+* GNSS socket no support stopping a stopped GPS module to apply delete mask.
```

Comment:
I'm confused with the first part, shouldn't it be "GNSS socket no**w** soupports"?

Reply:
yes, fixed to "GNSS socket now supports stopping a stopped GPS module to apply delete mask."